cellxgene_resources/ref_files/chry_regions.py

```diff
@@ -0,0 +1,573 @@
+regions = {
+    "NONPAR": [
+    "RNU6-1334P",
```

Comment:
IDs are needed for the actual QA because a pre-submission Dataset will not have symbols.
So if the symbols are listed in this file, the QA will need to do some work to get from symbol-->ID
Therefore, I think it just makes sense to store the Ensembl IDs